gematria/granite/convert_gb_token_model_to_tflite.sh

```diff
@@ -37,6 +37,8 @@ function print_error_and_exit() {
 # Parse command-line flags.
 # TODO(ondrasej): Consider using getopt instead of parsing the flags manually.
 gematria_export_as_seq2seq=0
+gematria_export_with_deltas=0
```

Comment:
IIRC, seq2seq and deltas are the same things under two slightly different names (which is, sadly, confusing).

The way the model is designed, the graph builder models should just always be seq2seq (because the non-seq2seq version doesn't work great).

Reply:
Got it. I'll combine the two flags (but keep support for seq2num around, i.e. keep seq2seq behind a flag as opposed to it being the only option).